front/src/modules/trainschedule/components/TimetableV2/TimetableToolbar.tsx

```diff
@@ -143,6 +170,15 @@ const TimetableToolbar = ({
               >
                 <Trash />
               </button>
+              <button
+                aria-label={t('timetable.deleteSelection')}
```

Comment:
```suggestion
                aria-label={t('timetable.downloadSelection')}
```

You will also have to add downloadSelection translation in `/public/locales/en`(and fr)`/operationalStudies/scenario.json /timetable`

Reply:
Done